legacy/build-deploy-docker-compose.sh

```diff
@@ -668,6 +679,11 @@ if [[ "$BUILD_TYPE" == "pullrequest"  ||  "$BUILD_TYPE" == "branch" ]]; then
   done
 fi
 
+if [ "$(featureFlag DOCKER_PROJECT_BUILDER)" = enabled ]; then  
+  echo "stopping builder $DOCKER_PROJECT_BUILDER"
+  docker builder stop --builder $DOCKER_PROJECT_BUILDER 
+fi  
```

Comment:
what if im running builds across two environments in the same project and my first build stops the builder while the other is still building?

Reply:
This isn't much of a concern considering the failure of being able to share the project builder